src/components/search-toolbar/SearchToolbar.jsx

```diff
@@ -0,0 +1,111 @@
+import { useState, useEffect } from 'react'
+import { Box, Typography, Grid, TextField } from '@mui/material'
+import ArrowBackIcon from '@mui/icons-material/ArrowBack'
+import SearchIcon from '@mui/icons-material/Search'
+import { styles } from '~/components/search-toolbar/SearchToolbar.styles'
+import TitleWithDescription from '../title-with-description/TitleWithDescription'
+import AsyncAutocomplete from '~/components/async-autocomlete/AsyncAutocomplete'
+import getEmptyArrayData from '~/utils/get-empty-array-data'
+import AppButton from '../app-button/AppButton'
+import HashLink from '../hash-link/HashLink'
+import { useTranslation } from 'react-i18next'
```

Comment:
I think we can move it upper (import libraries first, then custom components/hooks), but it's not necessarily.